rocAL/rocAL_pybind/example/coco_pipeline.py

```diff
@@ -276,8 +277,8 @@ def draw_patches(img,idx, bboxes):
         thickness = 2
         image = cv2.UMat(image).get()
         image = cv2.rectangle(image, (int(loc_[0]*wtot ),int( loc_[1] *htot)),(int((loc_[2] *wtot) ) ,int((loc_[3] *htot) )) , color, thickness)  
-        #cv2.imwrite(str(idx)+"_"+"train"+".png", 255*image)
-        cv2.imwrite(str(idx)+"_"+"train"+".png", image)
+        cv2.imwrite(str(idx)+"_"+"train"+".png", 255*image)
```

Comment:
this is not a correct inverse normalization value for the image. The output was washed out when displayed.